orchagent/aclorch.cpp

```diff
@@ -349,6 +361,42 @@ static acl_table_action_list_lookup_t defaultAclActionList =
                 }
             }
         }
+    },
+    {
+        // MARK_META
+        TABLE_TYPE_MARK_META,
+        {
+            {
+                ACL_STAGE_INGRESS,
+                {
+                    SAI_ACL_ACTION_TYPE_PACKET_ACTION
+                }
+            }
+        }
+    },
+    {
+        // MARK_METAV6
+        TABLE_TYPE_MARK_META_V6,
```

Comment:
Is it possible to combine the table `TABLE_TYPE_MARK_META` and `TABLE_TYPE_MARK_META_V6`?

Reply:
we can do it but that would result increating a bigger table which would be wasteful. Currently only broadcom supports the merged table as per my understanding. We need to have all the match attributes which are necessary for Ipv4/v6 packet matching.